packages/browser/package-lock.json

```diff
@@ -9,8 +9,6 @@
       "version": "1.8.1",
       "license": "MIT",
       "dependencies": {
-        "@inrupt/oidc-client-ext": "^1.8.1",
```

Comment:
I don't understand how this leads to changes in the lockfile of `-browser`? Shouldn't it still be depending on the stable version of `-oidc` that does not yet depend on `-core`? (I really don't know what Lerna is doing, but I feel like it messes up the lockfiles continuously and practically makes them worthless :/ )

Reply:
It's a bit worrying, but I don't know either. I did run `npm install` in the oidc module, and then `npm ci` at the top level, which triggers `lerna bootstrap`, and I think that may be what messes up the `package-lock`, but I'm not sure, nor can I tell if that's the right thing to do. I'll dig into separately.